### PR TITLE
[Refactor] Add some comments

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: gensrc/thrift/PlanNodes.thrift [hunk 1]
```diff
@@ -420,6 +420,7 @@ struct TSchemaScanNode {
   11: optional Types.TUserIdentity current_user_ident   // to replace the user and user_ip
 }
 
+// If you find yourself changing this code, see also TLakeScanNode
```
Comment (wyb) [inline gensrc/thrift/PlanNodes.thrift:423]: better to add some comment in `OlapScanNode.toThrift` function as well

Comment (sduzh) [reply gensrc/thrift/PlanNodes.thrift:423]: Updated, PTAL

**Review by wyb — APPROVED** [review]

**Review by ABingHuang — APPROVED** [review]
LGTM

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**